test/models/candlepin/repository_mapper_test.rb

```diff
@@ -33,6 +33,13 @@ def test_unprotected_suse
       assert_equal mapper.unprotected?, false
     end
 
+    def test_disconnected_no_http
```

Comment:
This test doesn't actually execute the changed method

Reply:
If I change the disconnected setting to try I get a vcr error though

Reply:
Take a look at the stack trace and see where from / why the HTTP request is going out. Maybe there's another bug?

Reply:
Oh I see, I need to call the `validate!` method in my test. Sorry after reading your comment and looking at it again, I see what needs updated.

Reply:
The test updates lgtm. If @lfu already tested your changes, I am ok acking

Reply:
@parthaa she did here: https://github.com/Katello/katello/pull/9484#issuecomment-885687772